dapp/src/routes/DnsVerification/DnsVerification.svelte

```diff
@@ -0,0 +1,236 @@
+    <div
+      class="flex flex-col mb-4 transition-all ease-in-out duration-500 bg-white p-10 rounded-lg dropshadow-default"
+    >
+      <div
+        class="mb-4 text-2xl text-left font-bold body flex flex-row items-center"
+      >
+        <div class="mr-3">DNS Verification</div>
+      </div>
+      <div class="body">
+        {newDisplay('dns').description}
+      </div>
+    </div>
+
+    <VerificationStep
+      step={1}
+      bind:currentStep
+      title="Enter Web Domain Name"
+      description="Enter the domain you want want to prove ownership of."
+    >
+      <div class="flex w-full mt-8">
+        <Input
+          placeholder="Enter web domain name"
+          class="mr-8"
+          bind:value={domainUrl}
+          disabled={currentStep !== 1}
+          name="enter-domain"
+        />
+        {#if currentStep === 1}
+          <PrimaryButton
+            text="Submit"
+            onClick={async () => {
+              next(async () => {
+                try {
+                  let isValid = isValidUrl(domainUrl);
+
+                  if (!isValid) {
+                    throw new Error('Invalid domain');
+                  }
+                  dnsClaim = getPreparedUnsignedDnsMessage(
+                    domainUrl,
+                    $userData
+                  );
+                } catch (err) {
+                  alert.set({
+                    variant: 'error',
+                    message: `Failed to create DNS claim: ${
+                      err?.message || JSON.stringify(err)
+                    }`,
+                  });
+                  throw new Error(err.message);
+                }
+              });
+            }}
+            class="ml-4 lg:ml-0"
+            small
+          />
+        {/if}
+      </div>
+    </VerificationStep>
+
+    <VerificationStep
+      step={2}
+      bind:currentStep
+      title="Sign Prompt"
+      description="Sign the message presented to you containing your domain."
+    >
+      {#if currentStep >= 2}
+        <div class="flex items-center w-full py-2 mt-8">
+          <textarea
+            class="overflow-x-auto rounded-lg bg-gray-100 body p-2 mr-4 w-full resize-none"
+            bind:value={dnsClaim}
+            readonly
+            disabled
+          />
+          <CopyButton text={dnsClaim} />
+        </div>
+      {/if}
+      {#if currentStep === 2}
+        <PrimaryButton
+          text="Sign Message"
+          class="mt-8 lg:w-48"
+          onClick={async () => {
+            next(async () => {
+              dnsMessage = await getSignedDnsClaim(
+                $userData,
+                domainUrl,
+                $wallet
+              );
+            });
+          }}
+          disabled={lock}
+        />
+      {/if}
+    </VerificationStep>
+
+    <VerificationStep step={3} bind:currentStep title="Upload TXT Record">
+      <div class="body">
+        In your DNS settings for your website, add a new TXT record for @ and
```

Comment:
I don't think we should phrase the DNS verification to be solely about websites.